### PR TITLE
feat: build single container

### LINKED ISSUE #1261: Build single container function in the proj.sh
See discussion https://nbisweden.slack.com/archives/C01D13EKUV7/p1670339002013859

### PR DESCRIPTION
**Pull request by inghylt**
**Related issue(s) and PR(s)**  
This PR closes #1261

**Type of change**  

- [x] New feature (non-breaking change which adds functionality)
 
**List of changes made**  

Add a new function to proj.sh

**Screenshot of the fix**  

**Testing**  

Instructions on how to test
- `source proj.sh`
- test `build-specific` with single and multiple arguments

**Further comments**  

I wasn't sure whether I should use `container(s)` or `service(s)` in the `Available commands` section, but I went with `container(s)` since that is used in other commands 

**Definition of Done checklist**  
- [x] My code meets the Acceptance Criteria
- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code and commented any hard-to-understand areas
- [x] Tests and lint/format validations are passing
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by e0** [comment]
@inghylt This seems to work well! I think the branch may not have been based on the `main` one though. I know that GitHub doesn't update the diff properly sometimes so not sure if this would add duplicate commits. Could we update this PR? I think either by changing the target branch to something else and back to `main`, or force pushing a clean version of the branch based off `main`.

File: proj.sh [hunk 1]
```diff
@@ -23,6 +23,10 @@ function build-stack {
   docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-local.yml build
 }
 
+function build-specific {
+  docker-compose --env-file env-local.env -f docker-compose.yml -f docker-compose-local.yml up -d --no-deps --build $@
```
Comment (MalinAhlberg) [inline proj.sh:27]:
```suggestion
  docker compose --env-file env-local.env -f docker-compose.yml -f docker-compose-local.yml up -d --no-deps --build $@
```

We use `docker compose`  on all other places in this file, and it's also the only alternative that works for on my setup :)

**Review by MalinAhlberg — COMMENTED** [review]
Good work, it's very convenient to have this! Also nice that you can build multiple containers at once!

I have one comment though, and I'm ready to approve when that's fixed.

**Comment by inghylt** [comment]
Closing this in favor of #1274

**Pull request closed without merging**